src/styleguide.md

```diff
@@ -4,31 +4,260 @@ title: Styleguide - Way2Web Design System
+        <span class="flex-1 bg-lightShade"></span>
+      </div>
+    </div>
+    <div>
+      <span>--way-color-light-tint</span>
+      <div class="flex flex-col h-8 overflow-hidden rounded">
+        <span class="flex-1 bg-lightTint"></span>
+      </div>
     </div>
   </div>
-  <div class="w-1/2 p-4 md:1/6 sm:w-1/3">
-    <span>Blue-900</span>
-    <div class="flex flex-col h-32 overflow-hidden rounded">
-      <span class="flex-1 bg-blue-900"></span>
+  
+  <div>
+    <legend class="mb-2 font-bold border-b">Medium</legend>
+    <div>
+      <span>--way-color-medium</span>
+      <div class="flex flex-col h-8 overflow-hidden rounded">
+        <span class="flex-1 bg-medium"></span>
+      </div>
+    </div>
+    <div>
+      <span>--way-color-medium-contrast</span>
+      <div class="flex flex-col h-8 overflow-hidden rounded">
+        <span class="flex-1 border bg-mediumContrast"></span>
+      </div>
+    </div>
+    <div>
+      <span>--way-color-medium-shade</span>
+      <div class="flex flex-col h-8 overflow-hidden rounded">
+        <span class="flex-1 bg-mediumShade"></span>
+      </div>
+    </div>
+    <div>
+      <span>--way-color-medium-tint</span>
+      <div class="flex flex-col h-8 overflow-hidden rounded">
+        <span class="flex-1 bg-mediumTint"></span>
+      </div>
     </div>
   </div>
+
+  <div>
+    <legend class="mb-2 font-bold border-b">Dark</legend>
+    <div>
+      <span>--way-color-dark</span>
+      <div class="flex flex-col h-8 overflow-hidden rounded">
+        <span class="flex-1 bg-dark"></span>
+      </div>
+    </div>
+    <div>
+      <span>--way-color-dark-contrast</span>
+      <div class="flex flex-col h-8 overflow-hidden rounded">
+        <span class="flex-1 border bg-darkContrast"></span>
+      </div>
+    </div>
+    <div>
+      <span>--way-color-dark-shade</span>
+      <div class="flex flex-col h-8 overflow-hidden rounded">
+        <span class="flex-1 bg-darkShade"></span>
+      </div>
+    </div>
+    <div>
+      <span>--way-color-dark-tint</span>
+      <div class="flex flex-col h-8 overflow-hidden rounded">
+        <span class="flex-1 bg-darkTint"></span>
+      </div>
+    </div>
+  </div>
+</div>
+
 </div>
+
+### Typography tokens
+
+<div class="block p-8 mb-5 bg-white rounded-lg shadow-lg">
+  <div class="mb-12">
+    <legend class="mb-2 font-bold border-b">
+      Font-family
+    </legend>
+    <div class="flex items-center mb-8">
+        <span class="mr-3 w-72">--way-font-family</span>
+        <span>
+          -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif
+        </span>
+    </div>
+  </div>
+
+  <div class="mb-12">
+    <legend class="mb-2 font-bold border-b">
+      Font-sizes
+    </legend>
+    <div class="flex items-center mb-8">
+        <span class="mr-3 w-72">--way-font-size-xx-small (10px)</span>
+        <span class="text-xxSmall">The quick brown fox...</span>
```

Comment:
**suggestion**: I would make this the full sentence: "The quick brown fox jumps over the lazy dog". Because it contains all of the letters of the English alphabet.